### PR TITLE
Refactor reminder tests & update documentation (UG/PPP)

### LINKED ISSUE #165: [PE-D][Tester E] Unclear what each status tag is to be used for
Maybe it should be made clearer that tagging something as "interview" or otherwise does not actually imply that it actually has an interview associated in the right panel.

This confused me greatly when I was testing `sort r/interview`.

-------------
Labels: `severity.Low` `type.DocumentationBug`
original: waynezsy/ped#6

### PR DESCRIPTION
**Pull request by jolynloh**
- [x] Refactor reminder tests
- [x] Update UG for clarity
- [x] PPP draft

### CODE REVIEW

**Review by nmtuan2001 — COMMENTED** [review]
LGTM!

File: src/test/java/seedu/application/logic/commands/RemindCommandTest.java [hunk 1]
```diff
@@ -20,24 +21,37 @@
...
-        assertCommandSuccess(command, model, expectedCommandResult, expectedModel);
-        assertEquals(Collections.emptyList(), model.getApplicationsWithUpcomingInterviewList());
+        RemindCommand command = new RemindCommand();
+        assertCommandSuccess(command, localModel, expectedCommandResult, localExpectedModel);
+        assertEquals(Collections.emptyList(), localModel.getApplicationsWithUpcomingInterviewList());
     }
 
     @Test
-    public void execute_oneUpcomingInterview() {
+    public void execute_oneUpcomingInterviews() {
```
Comment (teekaytai) [inline src/test/java/seedu/application/logic/commands/RemindCommandTest.java]:
```suggestion
    public void execute_oneUpcomingInterview() {
```

**Review by teekaytai — COMMENTED** [review]
LGTM too! Just happened to notice one very minor typo

File: src/test/java/seedu/application/logic/commands/RemindCommandTest.java (hunk 1 above)
Comment (jolynloh) [reply src/test/java/seedu/application/logic/commands/RemindCommandTest.java]: thanks for spotting! 😄

**Pull request merged**